<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptWidget</class>
 <widget class="QWidget" name="ScriptWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>460</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="toolTip">
      <string>Script status / progress</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>Idle</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="toolButtonRun">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Run this script</string>
     </property>
     <property name="icon">
      <iconset resource="qchdman.qrc">
       <normaloff>:/images/run.png</normaloff>:/images/run.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonStop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Terminate the running script</string>
     </property>
     <property name="icon">
      <iconset resource="qchdman.qrc">
       <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButtonActions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Tools menu</string>
     </property>
     <property name="icon">
      <iconset resource="qchdman.qrc">
       <normaloff>:/images/tools.png</normaloff>:/images/tools.png</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPlainTextEdit" name="plainTextEditScript">
      <property name="toolTip">
       <string>Script editor</string>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabScriptLog">
       <attribute name="title">
        <string>Script log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayoutScriptLog">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEditLog">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabProjectMonitor">
       <attribute name="title">
        <string>Project monitor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayoutProjectMonitor">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidgetProjectMonitor">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Project ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Progress</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Command</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolButtonRun</tabstop>
  <tabstop>toolButtonStop</tabstop>
  <tabstop>toolButtonActions</tabstop>
  <tabstop>plainTextEditScript</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>plainTextEditLog</tabstop>
  <tabstop>treeWidgetProjectMonitor</tabstop>
 </tabstops>
 <resources>
  <include location="qchdman.qrc"/>
 </resources>
 <connections/>
</ui>
